### PR TITLE
Users/yuliang/server

### PR DESCRIPTION
**Pull request by wish99benjamin**
### Summary
- Update Api interface:
    - topicName(for calling API): a new paramater with boolean type that specify if llm-server needs to generate conversation topic
    - topicName(for return value): conversation topic(string) | undefined(if topicName != true)
    - test.html for usage example
- Modify prompts:
    - remove export statement, add render statement
    - remove import statement
    - modify package usage
- Delete Unused Componenet "LLM"

### Screenshot
upload the screenshot of the changes made

### Checklist
- [x] My code follows ESLint rules and passes all tests.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: insta-ui/src/server/prompts.ts [hunk 1]
```diff
@@ -43,5 +44,59 @@ export function generateMessages(textInput: string, imageInput?: string, previou
...
+//         { type: 'text', text: `Revise the provided code: ${code}` }
+//     ];
+
+//     messages.push({
+//         role: 'user',
+//         content: userContent
+//     });
+
+//     return messages;
+// }
```
Comment (junkaiman) [inline insta-ui/src/server/prompts.ts]: You can safely delete the comments to keep the codebase clean :) We can always retrieve it from git history

Comment (wish99benjamin) [reply insta-ui/src/server/prompts.ts]: Sure, I'll delete them

**Review by junkaiman — DISMISSED** [review]
looks good. you can run `npm run build` locally to fix the linting issues locally and then push the latest update here

**Comment by wish99benjamin** [comment]
Unused Componenet "LLM" is deleted since it causes lint to fail after API is updated

**Review by junkaiman — APPROVED** [review]

**Pull request merged**